critiquebrainz/db/users.py

```diff
@@ -0,0 +1,17 @@
+from critiquebrainz import db
+from critiquebrainz.db import exceptions as db_exceptions
+import sqlalchemy
+
+
+def getusers_id(usernames):
```

Comment:
Yeah, at the beginning it was returning only ids, I changed it and forgot to update.